res/layout/edit_box_dialog.xml

```diff
@@ -28,7 +28,7 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:ems="10" 
-		android:inputType="textNoSuggestions"
+		android:inputType="textNoSuggestions|textCapSentences"
 		>
```

Comment:
Looks good to me, I would shorten the XML using /> instead of a complete closing tag though :+1: